src/openms/include/OpenMS/ANALYSIS/OPENSWATH/SpectrumExtractor.h

```diff
@@ -110,64 +118,129 @@ namespace OpenMS
 
     void getDefaultParameters(Param& params);
 
+    /**
+      Picks a spectrum's peaks and saves them in picked_spectrum.
+      The spectrum is first smoothed with a Gaussian filter (default) or using the
+      Savitzky-Golay method.
+      For these filters it is possible to set parameters like gauss_width_,
+      sgolay_frame_length_ and sgolay_polynomial_order_ through their setters.
+      The peak picking is executed with PeakPickerHiRes. It's possible to set the
+      signal_to_noise_ parameter.
+      @param[in] spectrum The input spectrum
```

Comment:
There should also be a line break between the @brief, longer description, and parameters.